modules/op-resolver-http.adoc

```diff
@@ -0,0 +1,93 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="resolver-http-specify_{context}"]
+= Specifying a remote pipeline or task with the HTTP Resolver
+
+When creating a pipeline run or task run, you can specify a remote pipeline or task from an HTTP or HTTPS URL.
+
+.Procedure
+
+* To specify a remote pipeline or task from an HTTP URL, use the following format in the `pipelineRef` or `taskRef` spec:
```

Comment:
Same comment as the other module - if this step is mandatory, I'd remove the "To" at the beginning to simply instruct users directly to take this action:
```suggestion
* Specify a remote pipeline or task from an HTTP URL, using the following format in the `pipelineRef` or `taskRef` spec:
```